index.html

```diff
@@ -1,42 +1,85 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <link rel="stylesheet" href="style.css">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name=" viewport" content="width=device-width, initial-scale=1.0">
-  <title>Portfolio Skeleton</title>
-</head>
-<body>
-  <div id="mobile-background">
-    
-    
-    <header>
-      <h1>Inyambo</h1>  
-    
-      
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;300;400;500;600;700&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css"
+      integrity="sha512-KfkfwYDsLkIlwQp6LFnl8zNdLGxu9YAA1QvwINks4PhcElQSvqcyVLLD9aMhXd13uQjoXtEKNosOWaZqXgel0g=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <link rel="stylesheet" href="style.css" />
+
+    <title>Portfolio Skeleton</title>
+  </head>
+  <body>
+    <header class="header-section">
+      <div class="mobile-menu">
+        <img
+          src="ClosingIcon.png"
+          class="closing-icon"
+          alt="closed icon"
+        />
+        <ul class="right-sidebar">
+          <li class="menu-li">
+            <a href="#portfolio" class="nav-link">Portfolio</a>
+          </li>
+          <li class="menu-li"><a href="#about" class="nav-link">About</a></li>
+          <li class="menu-li">
+            <a href="#contact" class="nav-link">Contact</a>
+          </li>
+        </ul>
+      </div>
+
+      <a class="header-logo" href="#">
+        <div>
+          <h2 class="my-logo">Inyambo</h2>
+        </div>
+      </a>
       <nav>
-        <div id="Menu-bar"></div>
+        <div class="nav-right">
+          <img src="Images/Enabled.png" alt="normal button" />
```

Comment:
Kindly fix the URL of this image as it is not being displayed correctly in your Mobile view 😄

![image](https://user-images.githubusercontent.com/73714615/186907715-42695922-0c79-41f9-a34b-26f0b917849f.png)